.buildkite/pipeline.yml

```diff
@@ -1,18 +1,38 @@
 steps:
-  - label: "cargo test"
+  - label: "cargo test integration-tests*"
     command: |
       source ~/.cargo/env && set -eux
-      RUST_BACKTRACE=1 RUST_LOG=WARN cargo test --locked --workspace
+      RUST_BACKTRACE=1 RUST_LOG=WARN cargo test --locked -p 'integration-tests' -j 16
```

Comment:
Why do we need `-j 16` argument here? Cargo runs tests in parallel by default.

Reply:
@matklad Yes, I'm going to remove it; I think it doesn't make any difference. 

Anyway, I want to do this one next, as it's a clear win: https://github.com/near/nearcore/pull/5771